include/co/def.h

```diff
@@ -47,19 +47,22 @@ typedef uint64_t uint64;
 #define  unlikely(x)  (x)
 #endif
 
-#ifdef _MSC_VER
 #include "config.h"
-#endif
 
 // __codef: define global variables
 // __codec: declare classes, functions, global variables
-#if defined(_MSC_VER) && USING_CO_DLL > 0
-  #ifdef BUILDING_CO_DLL
-    #define __codef __declspec(dllexport)
-    #define __codec __declspec(dllexport)
+#if COCOYAXI_SHARED > 0
+  #ifdef _WIN32

```

Comment:
@SpaceIm  Does it work for mingw here?

Reply:
There should be no problem if mingw also supports `__declspec(dllexport)` and `__declspec(dllimport)`.